### PR TITLE
Adding preview 2 links to .NET Core Linux Prerequisites

### PR DESCRIPTION
**Pull request by JRAlexander**
Updating .NET Core Linux Prerequisites with .NET Core 2.1 preview 2 links.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?branch=pr-en-us-4951&tabs=netcore2x)

### CODE REVIEW

**Comment by JRAlexander** [comment]
Closing and reopening

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -351,8 +356,10 @@ To install .NET Core 2.x for supported SUSE Linux Enterprise Server and OpenSUSE
 >[!IMPORTANT]
 > To use .NET Core 2.1 with Visual Studio, you need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/preview).
 
-* .NET Core Runtime 2.1.0-preview1 [install link](https://www.microsoft.com/net/download/linux-package-manager/opensuse/runtime-2.1.0-preview1)
-* .NET Core SDK 2.1.300-preview1 [install link](https://www.microsoft.com/net/download/linux-package-manager/opensuse/sdk-2.1.300-preview1)
+* [.NET Core Runtime 2.1.0-preview2 install link](https://www.microsoft.com/net/download/linux-package-manager/opensuse/runtime-2.1.0-preview2)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: this is not following the style of the other links

**Review by mairaw — APPROVED** [review]
LGTM. Left a comment and also noticed some Linux distros didn't have a preview2 link. By design?

File: docs/core/linux-prerequisites.md (hunk 1 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Fixed it! Thanks, @mairaw!

**Pull request merged**